Common/Src/can_driver.c

```diff
@@ -0,0 +1,84 @@
+#include "can_driver.h"
+
+uint32_t tx_mailbox;
+
+CAN_Frame_t CAN_frame_init(CAN_HandleTypeDef* handler, uint32_t id) {
+    // default conditions
+    CAN_Frame_t ret = {
+        .hcan = handler,
+        .id_type = CAN_ID_STD,
+        .rtr = CAN_RTR_DATA,
+        .data_length = 8,
```

Comment:
Not sure if this is what you intend, but if the goal here was to set it to the max it can hold, then setting it to `MAX_BYTES` might be better for readability.